### ISSUE TITLE
Flake8 cleanup and typo fixes for workers

### ISSUE THREAD

**Issue by rtnpro** [CentOS/container-pipeline-service#85]
There's a bunch of linting errors and typos in the workers. This needs to be fixed ASAP before doing further changes. As an Open Source project, we need to agree on certain coding styleguide and best practices, so that everyone writes clean, readable code.

**Comment by kbsingh** [comment]
good point to bring through to the next planning meeting. I think step 1 really is to write better code and do hinting around others works in their PRs when the PRs are raised.

the retrospective work into existing base should be addressed in the relevant sprints where we can take on board the backlog work ( so ensure there is a card created and tracked for this )

**Comment by navidshaikh** [comment]
Most of the code is cleaned with mentioned issues. Will close this once we put pep8 flake8 etc gates directly in CI to ensure no violating code is in place.
